source/DataAccess/Oracle/OracleDDLHelper.ts

```diff
@@ -3,24 +3,26 @@ import { injectable } from 'inversify';
 
 @injectable()
 export default class OracleDDLHelper implements IDDLHelper {
```

Comment:
This class still feels weird and could be called 'test baggage' but I can foresee a future where DDL 'query' for a database backend will certainly differ when compared to a REST service. IDDLHelper would likely need to be made generic and extended in various ways. Again - for now it's just for generating a query in the form of a query and built to be testable/injectable.

Reply:
Good thoughts. This seems fine for now, though your points make sense.